x/cert/keeper/querier.go

```diff
@@ -138,7 +138,7 @@ func queryPlatform(ctx sdk.Context, path []string, keeper Keeper, legacyQuerierC
 		return nil, nil
 	}
 
-	res, err2 := codec.MarshalJSONIndent(legacyQuerierCdc, types.QueryPlatformResponse{Platform: platform})
+	res, err2 := codec.MarshalJSONIndent(legacyQuerierCdc, platform)
```

Comment:
Minor but `err1`, `err2` etc are usually a smell. Please use `err` throughout each scope whenever possible.

Reply:
It's fixed.